code/game/objects/items/storage/uplink_kits.dm

```diff
@@ -205,6 +205,20 @@
 			new /obj/item/dnainjector/cryokinesis(src)
 			new /obj/item/gun/energy/temperature/pin(src)
 			new /obj/item/melee/transforming/energy/sword/saber/blue(src) // 8 tc //see see it fits the theme bc its blue and ice is blue
+
+		if("anime") //I think around 31-33
+			new /obj/item/clothing/suit/armor/vest/leather(src) // ~1 tc
+			new /obj/item/clothing/head/anime_fedora(src) // ~1-3 tc? has the 30 throw damage of the null rod version and a very high throw speed
+			new /obj/item/clothing/shoes/sneakers/black(src) // 0 tc
+			new /obj/item/choice_beacon/hacked_anime(src) // unique item however it has almost 0 non joke uses
+			new /obj/item/choice_beacon/hacked_anime(src)
+			new /obj/item/choice_beacon/hacked_anime(src)
+			new /obj/item/clothing/neck/tie/horrible(src) // 0 tc
+			new /obj/item/guardiancreator/rare(src) // 18-20 tc
+			new /obj/item/book/granter/spell/blinkdagger(src) //~10 tc, hard to tell
```

Comment:
By itself? This is dual-esword levels of power.
I'd place it at a full 18 TC of value.